VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -202,49 +170,24 @@ class VungleAdapter : PartnerAdapter {
                 .trim()
                 .takeIf { it.isNotEmpty() }
                 ?.let { appId ->
-                    Vungle.init(appId, context.applicationContext, object : InitCallback {
+                    VungleAds.init(context, appId, object : InitializationListener {
                         override fun onSuccess() {
-                            Plugin.addWrapperInfo(
-                                VungleApiClient.WrapperFramework.vunglehbs,
+                            VungleAds.setIntegrationName(
+                                VungleAds.WrapperFramework.vunglehbs,
                                 adapterVersion
                             )
 
-                            resumeOnce(
-                                Result.success(
-                                    PartnerLogController.log(
-                                        SETUP_SUCCEEDED
-                                    )
-                                )
-                            )
+                            resumeOnce(Result.success(PartnerLogController.log(SETUP_SUCCEEDED)))
                         }
 
-                        override fun onError(exception: VungleException) {
-                            PartnerLogController.log(SETUP_FAILED, "Error: $exception")
-
-                            resumeOnce(
-                                Result.failure(
-                                    ChartboostMediationAdException(
-                                        getChartboostMediationError(
-                                            exception
-                                        )
-                                    )
-                                )
-                            )
+                        override fun onError(vungleError: VungleError) {
+                            PartnerLogController.log(SETUP_FAILED, "Error: $vungleError")
+                            resumeOnce(Result.failure(ChartboostMediationAdException(getChartboostMediationError(vungleError))))
```

Comment:
There are four of these where the should be formatted for readability

Reply:
Weird that the manually triggering doesn't auto do it. 🤔